sysutils/port-whatsnew/Portfile

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                port-whatsnew
+version             0.1
+revision            0
+categories          sysutils macports
+maintainers         gwmail.gwu.edu:egall openmaintainer
+platforms           darwin
+license             BSD
+supported_archs     noarch
+
+description         Show the svn log of what changed since this port \
+                    was installed.
+long_description    ${description}
+homepage            http://svn.macports.org/repository/macports/contrib/${name}
```

Comment:
Is this really the best possible place for this to live? Maybe submit a commit to put the sources under git? And what is this thing actually? svn hasn't been where macports lives in a long long time?

Reply:
@pmetzger see PR #20951 ; these are both addressing 11+ year old Trac tickets by adding some script that hasn't been updated much since then it appears. I am not in favor of adding these to MacPorts. The few (only?) person who might want to use this can just download it from the homepage listed in the Portfiles in my humble opinion.

Reply:
> these are both addressing 11+ year old Trac tickets

Not just any Trac tickets, Trac tickets that I opened and never gave up on over that time period. Age shouldn't matter. An issue is an issue no matter how old. 

> The few (only?) person who might want to use this can just download it from the homepage listed in the Portfiles in my humble opinion.

That's not as easily discoverable, and doesn't install the script into the MacPorts prefix though. Having a port to easily install these scripts is much more convenient than having to clone a separate repo, find the script you're interested in, and then put it in the correct place.